android/app/src/main/java/updated/mysterium/vpn/ui/base/BaseViewModel.kt

```diff
@@ -132,4 +134,11 @@ class BaseViewModel(useCaseProvider: UseCaseProvider) : ViewModel() {
             }
         }
     }
+
+    fun getGateways() = liveDataResult {
+        paymentUseCase.getGateways().map { it.name }.toMutableList().apply {
+            remove("paypal")
```

Comment:
Let's change return type to List<Gateway> and will map it in chain
Also it would be better to add paypal to Gateway enum as well check not by name and hardcoded value, but by enum value